homeassistant/components/habitica/__init__.py

```diff
@@ -30,21 +23,17 @@
     EVENT_API_CALL_SUCCESS,
     SERVICE_API_CALL,
 )
-from .sensor import SENSORS_TYPES
+from .coordinator import HabitipyData
 
 _LOGGER = logging.getLogger(__name__)
 
 INSTANCE_SCHEMA = vol.All(
-    cv.deprecated(CONF_SENSORS),
     vol.Schema(
         {
             vol.Optional(CONF_URL, default=DEFAULT_URL): cv.url,
             vol.Optional(CONF_NAME): cv.string,
             vol.Required(CONF_API_USER): cv.string,
             vol.Required(CONF_API_KEY): cv.string,
-            vol.Optional(CONF_SENSORS, default=list(SENSORS_TYPES)): vol.All(
-                cv.ensure_list, vol.Unique(), [vol.In(list(SENSORS_TYPES))]
-            ),
```

Comment:
Let's not do this yet

Reply:
Oh these are the sensor types we've talked about before.

I think we should retain this logic and add a deprecation issue so we can remove this import flow in 6 months as it doesn't add more value right now

Reply:
I'll double check this

Reply:
This is the warning in the logs that appears if I setup habitica with sensors variable via configuration.yaml:

![image](https://github.com/home-assistant/core/assets/4445816/4b02246d-1cf2-4487-9d35-a73b7f1ed5d1)

Reply:
I have reverted it , reduced SENSOR_TYPES to a minimum, and added a deprecation issue for the yaml config